sdk/python/tests/integration/registration/test_cli.py

```diff
@@ -128,6 +127,33 @@ def make_feature_store_yaml(project, test_repo_config, repo_dir_name: PosixPath)
     return yaml.safe_dump(config_dict)
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize("test_repo_config", FULL_REPO_CONFIGS)
```

Comment:
so here, instead of all repo configs we should use only ones with `online_store: None`

Reply:
I don't really understand how test_cli.py is testing the configs that I added. Adding 3 tests to test_repo_config.py increased total passed tests by 3. However the ones I added in test_cli didn't affect the total number of tests. Is there a doc that I read about the testing structure?

Reply:
@mirayyuce here are our docs on tests: https://docs.feast.dev/how-to-guides/adding-or-reusing-tests